eats/edinburgh-uk.md

```diff
@@ -0,0 +1,6 @@
+### Edinburgh, Scotland Great Places to Eat
+* Fenwicks
+* The Witchery
+* The Dome
+* Bar One
```

Comment:
@EmilyUna65 is this the place?: [All Bar One](http://www.allbarone.co.uk/national-search/scotland-and-northern-ireland/all-bar-one-george-street-edinburgh)